package.json

```diff
@@ -50,7 +52,7 @@
     "lodash.isobject": "^3.0.2",
     "lodash.omit": "^4.4.1",
     "lodash.tonumber": "^4.0.3",
-    "tether": "^1.3.4"
+    "tether": "balloob/tether#7e0d48e4ac2162e251dc2b69d7c13e6c29a5667f"
```

Comment:
This is to point at a tether build that is using Browserify. [Repo here.](https://github.com/balloob/tether/tree/feature/browserify-and-test) 

We can fork it to the Reactstrap org if preferred.